### ISSUE TITLE
Doubt about Loss.png 

### ISSUE THREAD

**Issue by appyfizzA** [crisb-DUT/DnCNN-tensorflow#17]
What is the x-axis in the file loss.png?
The y-axis is loss values, but x-axis isn't mentioned.

What is the loss value plotted against?

**Comment by crisb-DUT** [comment]
Hi @appyfizzA ,

It's iteration number.

Thanks

**Comment by appyfizzA** [comment]
Value sin x-axis are written as 2.000k, 4.000k and so on. What does the k stand for?

**Comment by crisb-DUT** [comment]
https://zh.wikipedia.org/zh-hans/%E5%8D%83_(%E5%89%8D%E7%B6%B4)

**Comment by appyfizzA** [comment]
@crisb-DUT 
so you run it for 2k/4k/6k/8k iterations? Meaning 2000/4000/6000/8000 iterations?
But the code runs for 50 epochs. 

And what is the relation between iteration and epoch in your code?

Please explain.

**Comment by lizhiyuanUSTC** [comment]
There are 953600 images in training data, and the batchsize is 128, which means 1 epoch needs 953600 / 128 iterations.

**Comment by appyfizzA** [comment]
Thanks @lizhiyuanUSTC 
When DATA_AUG = 4, the total no. of patches (each of size 40 by 40) is 954112. 
954112/128 = 7454, so 7454 iterations in one epoch.
Please confirm this.

However, DATA_AUG has been set to 1 in a recent update of generate_patches.py, right? 
So in that case, 238528 patches are created, and 238528/128 ~= 1863 iterations in one epoch ~= 8k iterations in 4 epochs.

please confirm if I'm right.

**Comment by crisb-DUT** [comment]
Hi @appyfizzA ,

For the configuration of generation_patches.py, plz refer to #18 

thanks
